docusaurus/docs/dev-docs/api/rest/guides/understanding-populate.md

```diff
@@ -0,0 +1,1653 @@
+              "provider_metadata": null,
+              "createdAt": "2021-11-23T14:05:33.460Z",
+              "updatedAt": "2021-11-23T14:05:46.084Z"
+            }
+          }
+        },
+        "blocks": [
+          {
+            "id": 2,
+            "__component": "blocks.related-articles"
+          },
+          {
+            "id": 2,
+            "__component": "blocks.cta-command-line",
+            "theme": "primary",
+            "title": "Want to give a try to a Strapi starter?",
+            "text": "❤️",
+            "commandLine": "git clone https://github.com/strapi/nextjs-corporate-starter.git"
+          }
+        ],
+        "seo": {
+          "id": 1,
+          "metaTitle": "Articles - FoodAdvisor",
+          "metaDescription": "Discover our articles about food, restaurants, bars and more! - FoodAdvisor",
+          "keywords": "food",
+          "metaRobots": null,
+          "structuredData": null,
+          "metaViewport": null,
+          "canonicalURL": null
+        },
+        "category": {
+          "data": {
+            "id": 4,
+            "attributes": {
+              "name": "European",
+              "slug": "european",
+              "createdAt": "2021-11-09T13:33:20.123Z",
+              "updatedAt": "2021-11-09T13:33:20.123Z"
+            }
+          }
+        },
+        "localizations": {
+          "data": [
+            {
+              "id": 10,
+              "attributes": {
+                "title": "Voici pourquoi il faut essayer la cuisine basque, selon un chef basque",
+                "slug": "voici-pourquoi-il-faut-essayer-la-cuisine-basque-selon-un-chef-basque",
+                "createdAt": "2021-11-18T13:33:19.948Z",
+                "updatedAt": "2023-06-02T10:57:19.606Z",
+                "publishedAt": "2022-09-22T13:00:00.069Z",
+                "locale": "fr-FR",
+                "ckeditor_content": // truncated content
+              }
+            }
+          ]
+        }
+      }
+    },
+    {
+      "id": 2,
+      // truncated content
+    },
+    {
+      "id": 3,
+      // truncated content
+    },
+    {
+      "id": 4,
+      // truncated content
+    }
+  ],
+  "meta": {
+    "pagination": {
+      "page": 1,
+      "pageSize": 25,
+      "pageCount": 1,
+      "total": 4
+    }
+  }
+}
+```
+
+</Response>
+</ApiCall>
+
+</SideBySideColumn>
+</SideBySideContainer>
+
+## Populate specific relations and fields
+
+You can also populate specific relations and fields, by explicitly defining what to populate. This requires that you know the name of fields and relations to populate.
+
+Relations and fields populated this way can be 1 or several level(s) deep. The following diagram compares data returned by the [FoodAdvisor](https://github.com/strapi/foodadvisor) example application when you populate [1 level deep](#populate-1-level-deep-for-specific-relations) vs. [2 levels deep](#populate-several-levels-deep-for-specific-relations):
```

Comment:
> level(s) deep

From an english side of things, `levels` is fine here since you mentioned "1 or" before it.

Reply:
Thanks! I wasn't sure and didn't take time to check 😅